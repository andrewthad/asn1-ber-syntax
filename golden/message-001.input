30220201 010402BB BBA01902 0405F5E1
00020100 02010030 0B300906 042B0681
01410109
