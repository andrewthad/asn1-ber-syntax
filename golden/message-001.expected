Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 3
            [ Value
                { tagClass = Universal
                , tagNumber = 2
                , contents = Integer 1
                }
            , Value
                { tagClass = Universal
                , tagNumber = 4
                , contents = OctetString
                    [ 0xbb
                    , 0xbb
                    ]
                }
            , Value
                { tagClass = ContextSpecific
                , tagNumber = 0
                , contents = Constructed
                    ( fromListN 4
                        [ Value
                            { tagClass = Universal
                            , tagNumber = 2
                            , contents = Integer 100000000
                            }
                        , Value
                            { tagClass = Universal
                            , tagNumber = 2
                            , contents = Integer 0
                            }
                        , Value
                            { tagClass = Universal
                            , tagNumber = 2
                            , contents = Integer 0
                            }
                        , Value
                            { tagClass = Universal
                            , tagNumber = 16
                            , contents = Constructed
                                ( fromListN 1
                                    [ Value
                                        { tagClass = Universal
                                        , tagNumber = 16
                                        , contents = Constructed
                                            ( fromListN 2
                                                [ Value
                                                    { tagClass = Universal
                                                    , tagNumber = 6
                                                    , contents = ObjectIdentifier
                                                        ( fromListN 4
                                                            [ 1
                                                            , 3
                                                            , 6
                                                            , 129
                                                            ]
                                                        )
                                                    }
                                                , Value
                                                    { tagClass = Application
                                                    , tagNumber = 1
                                                    , contents = Unresolved [ 0x09 ]
                                                    }
                                                ]
                                            )
                                        }
                                    ]
                                )
                            }
                        ]
                    )
                }
            ]
        )
    }
